e2e/spec/404.spec.e2e.ts

```diff
@@ -1,32 +1,49 @@
 Feature('404 error handling');
 
 //region 404 page for non-existing ILC route
-Scenario('Renders (SSR) global 404 page for non-existing ILC route', (I) => {
+Scenario('Renders (SSR) global 404 page for non-existing ILC route for domain "localhost:8233" (default)', (I, common: common) => {
     I.amOnPage('/nonexistent-path');
-    I.waitForText('404 not found', 10, 'body > div#body');
+    I.waitForElement('body > div#navbar > div.app-container > div.navbar-app', 30);
+    I.waitForText(common.text404NotFound, 10, 'body > div#body');
 });
-Scenario('Renders (CSR) global 404 page for non-existing ILC route', (I, peoplePage) => {
+Scenario('Renders (CSR) global 404 page for non-existing ILC route for domain "localhost:8233" (default)', (I, peoplePage, common: common) => {
     const notFoundPageLink = '#navbar a[href="/nosuchpath"]';
 
     I.amOnPage(peoplePage.peopleUrl);
     I.waitForElement(notFoundPageLink, 30);
     I.click(notFoundPageLink);
-    I.waitForText('404 not found', 10, 'body > div#body');
+    I.waitForElement('body > div#navbar > div.app-container > div.navbar-app', 30);
+    I.waitForText(common.text404NotFound, 10, 'body > div#body');
+});
+
+Scenario('Renders (SSR) global 404 page for non-existing ILC route for domain "127.0.0.1:8233"', (I, common: common) => {
+    I.amOnPage(common.url.localhostAsIPv4 + '/nonexistent-path');
+    I.waitForText(common.text404NotFound, 10, 'body > div#navbar'); // for "127.0.0.1:8233" inside of navbar will be rendered 404 fragment, like in body
```

Comment:
I think this test checks nothing... We need to validate that "other" 404 page can be rendered for custom domain.